### PR TITLE
Add GKE Sandbox to Kubernetes section

### PR DESCRIPTION
**Pull request by fvoznika**

### CODE REVIEW

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
++++
+title = "GKE Sandbox Quickstart"
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: I'm a bit wary of putting GKE docs on the gVisor website. Do you think this provides value over the ["Enabling GKE Sandbox"](https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods#enabling) and ["Running an application in a sandbox"](
https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods#sandboxed-application) sections in the GKE docs?

Maybe updating the GKE Sandbox docs to have a Service and allow verification by opening an outside connection would be better?

Comment (ianlewis) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Let's make a "tutorials" section and put this there. I'm worried about the expectation of maintaining this doc in the same format. Howtos or tutorials don't necessarily have that expectation and it's clearer what it's for.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+to the command, like below:
+
+```bash
+gcloud beta container node-pools create sandbox-pool --cluster=${CLUSTER_NAME} --image-type=cos_containerd --sandbox type=gvisor
+```
+
+If you prefer to use the console, select your cluster and select the **ADD NODE
+POOL** button:
+
+![+ ADD NODE POOL](/docs/user_guide/gke/node-pool-button.png)
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: This image doesn't exist?

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+Then select the **Image type** with **Containerd** and select **Enable sandbox
+with gVisor** option. Select other options as you like:
+
+![+ NODE POOL](/docs/user_guide/gke/add-node-pool.png)
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: This image doesn't exist?

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Extra new line

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+Now, let's deploy a WordPress site using GKE Sandbox. WordPress site requires
+two pods: web server in the frontend, MySQL database in the backend. Both
+applications use PersistentVolumes to store the site data data. 
+In addition, they use secret store to share MySQL password between them.
+
+First, let's download the deployment configuration files to add the runtime
+class annotation to them:
+
+```bash
+  curl -LO https://k8s.io/examples/application/wordpress/wordpress-deployment.yaml
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Do these commands need to have leading spaces?

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+  curl -LO https://k8s.io/examples/application/wordpress/mysql-deployment.yaml
+```
+
+Add **spec.template.spec.runtimeClassName** set to **gvisor** to both files,
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: s/Add/Add a/
s/like/as/

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+      tier: frontend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+    spec:
+      runtimeClassName: gvisor   <== ADD THIS LINE
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Make "ADD THIS LINE" a yaml comment

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+```bash
+  curl -LO https://k8s.io/examples/application/wordpress/wordpress-deployment.yaml
+  curl -LO https://k8s.io/examples/application/wordpress/mysql-deployment.yaml
+```
+
+Add **spec.template.spec.runtimeClassName** set to **gvisor** to both files,
+like shown below:
+
+```yaml
+# wordpress-deployment.yaml
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: I would put the file name outside the code block and make it bold. Something like

**wordpress-deployment.yaml:**

```yaml
# yaml here...
```

**mysql-deployment.yaml:**
```yaml
# yaml here...
```

That makes it easier to differentiate the files.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+        volumeMounts:
+        - name: mysql-persistent-storage
+          mountPath: /var/lib/mysql
+      volumes:
+      - name: mysql-persistent-storage
+        persistentVolumeClaim:
+          claimName: mysql-pv-claim
+```
+
+Note that apart from `runtimeClassName: gvisor`, nothing else is changed. Next,
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: s/nothing else is/nothing else about the Deployment has/

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+let's create a secret to store MySQL's password. Create a file called
+`secret.yaml` with the following content:
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Secrets have to have base64 encoded content. It's easier to have them create a secret like so

    kubectl create secret generic mysql-pass --from-literal=password=mysecretpassword

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+kubectl apply -f secret.yaml
+kubectl apply -f mysql-deployment.yaml
+kubectl apply -f wordpress-deployment.yaml
+```
+
+Wait for the deployments to be ready and an external IP to be assigned to the
+Wordpress service:
+
+```bash
+watch kubectl get service wordpress
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Maybe show example output with fake IP and an example of accessing the website.

    firefox "http://123.456.789.123"

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md [hunk 12]
```diff
@@ -0,0 +1,247 @@
...
+```
+
+Congratulations! You have just deployed a WordPress site using GKE Sandbox. Now,
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Add the "Congratulations!" part after they verify Wordpress in the browser.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 1 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Done.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 2 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Fixed.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 3 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Fixed.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 5 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Nope.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 8 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: I didn't like the space between lines, but I guess this is better than the comment.

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 10 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: Nice suggestion!

File: content/docs/user_guide/kubernetes/wordpress-quick-start.md (hunk 11 above)
Comment (fvoznika) [reply content/docs/user_guide/kubernetes/wordpress-quick-start.md]: I want to avoid using an specific browser.

**Comment by ianlewis** [comment]
Hmm, I thought you should have been able to see them. I fiddled with the settings so hopefully you should be able to read them now. IAM doesn't seem to allow us to make the build logs public.

```
Running ["HtmlCheck", "ScriptCheck", "LinkCheck", "ImageCheck"] on ["public/static"] on *.html... 

Ran on 23 files!

- public/static/docs/tutorials/docker/index.html
  *  internally linking to /docs/tutorials/wordpress/kubernetes/, which does not exist (line 477)
     <a href="/docs/tutorials/wordpress/kubernetes/">Learn how to deploy WordPress with Kubernetes</a>
- public/static/docs/user_guide/kubernetes/index.html
  *  internally linking to /docs/user_guide/tutorials/wordpress/kubernetes/, which does not exist (line 443)
     <a href="/docs/user_guide/tutorials/wordpress/kubernetes/">Here</a>
htmlproofer 3.10.2 | Error:  HTML-Proofer found 2 failures!
```

File: content/docs/user_guide/kubernetes.md [hunk 13]
```diff
@@ -20,7 +20,19 @@ use either the `io.kubernetes.cri.untrusted-workload` annotation or
...
+WordPress site. You can view the full documentation [here][gke-sandbox-docs].
+
 [containerd]: https://containerd.io/
 [minikube]: https://github.com/kubernetes/minikube/blob/master/deploy/addons/gvisor/README.md
+[gke]: https://cloud.google.com/kubernetes-engine/
+[gke-sandbox]: https://cloud.google.com/kubernetes-engine/sandbox/
+[gke-sandbox-docs]: https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods
 [gvisor-containerd-shim]: https://github.com/google/gvisor-containerd-shim
 [runtimeclass]: https://kubernetes.io/docs/concepts/containers/runtime-class/
+[wordpress-quick]: /docs/user_guide/tutorials/wordpress/kubernetes/
```
Comment (ianlewis) [inline content/docs/user_guide/kubernetes.md]: This URL seems to be incorrect

**Pull request merged**